app/src/androidTest/java/com/willowtree/vocable/tests/MainScreenTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.willowtree.vocable.tests
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.willowtree.vocable.screens.MainScreen
+import com.willowtree.vocable.utility.assertTextMatches
+import com.willowtree.vocable.utility.tap
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MainScreenTest : BaseTest() {
+
+    private val mainScreen = MainScreen()
+
+    @Test
+    fun verifyClickingPhraseUpdatesCurrentText() {
+        mainScreen.apply {
+            firstPhrase.tap()
+            currentText.assertTextMatches("Please")
+        }
+    }
+
+    @Test
+    fun verifyDefaultCategoriesExist() {
+        mainScreen.apply {
+            mainScreen.verifyDefaultCategories()
```

Comment:
I'd probs also call this verifyDefaultCategoriesExist() so that if it fails in there we still get an idea of what we're trying to verify in the ci logs.

Reply:
updated